src/MyPlot/EventListener.php

```diff
@@ -236,22 +237,29 @@ public function onPlayerMove(PlayerMoveEvent $event) {
 		if($event->isCancelled()) {
 			return;
 		}
+		if($event->getFrom()->floor()->equals($event->getTo()->floor())) return; // Save hell-a-lot of performance by ignoring movements on the same block, for example rotating the head
```

Comment:
It can not. Cancelling the actual event is more important than the performace check

Reply:
I think you’re missing my point. You have two if statements that if true will do the same thing. It would be better to have a single statement with both separated by an or